src/state/user/reducer.ts

```diff
@@ -131,6 +131,12 @@ const userSlice = createSlice({
     // for all existing users with a previous version of the state in their localStorage.
     // In order to avoid this, we need to set a default value for each new property manually during hydration.
     builder.addCase(updateVersion, (state) => {
+      // If `selectedWallet` is ConnectionType.UNI_WALLET (deprecated) switch it to ConnectionType.UNISWAP_WALLET
+      // @ts-ignore
```

Comment:
why the ts-ignore?

Reply:
React never expects `state.selectedWallet` to equal `UNIWALLET` so it throws an error

Reply:
because it is not a typed value of state.selectedWallet anymore.

This is good, but I'd like to see a more proper, segmented out migration codepath in the future. Can that be ticketed?

Reply:
cc: @JFrankfurt